knowledge-base/listview-cross-orientation-nesting.md

```diff
@@ -0,0 +1,241 @@
+                                       Grid.Row="0" />
+
+                                <telerik:RadListView x:Name="InnerListView"
+                                                     ItemsSource="{Binding SubItems}"
+                                                     BindingContextChanged="OnListViewBindingContextChanged"
+                                                     Grid.Row="1">
+                                    <telerik:RadListView.ItemTemplate>
+                                        <DataTemplate x:DataType="local:KanbanSubItem">
+                                            <telerik:ListViewTemplateCell>
+                                                <VerticalStackLayout Padding="10"
+                                                                     Spacing="5">
+                                                    <Label Text="{Binding Title}"
+                                                           FontAttributes="Bold" />
+                                                    <Label Text="{Binding Deadline, StringFormat='{0:g}'}" />
+                                                </VerticalStackLayout>
+                                            </telerik:ListViewTemplateCell>
+                                        </DataTemplate>
+                                    </telerik:RadListView.ItemTemplate>
+                                    <telerik:RadListView.LayoutDefinition>
+                                        <telerik:ListViewLinearLayout VerticalItemSpacing="5"
+                                                                      Orientation="Vertical" />
+                                    </telerik:RadListView.LayoutDefinition>
+                                </telerik:RadListView>
+                            </Grid>
+                        </telerik:ListViewTemplateCell>
+                    </DataTemplate>
+                </telerik:RadListView.ItemTemplate>
+                <telerik:RadListView.LayoutDefinition>
+                    <telerik:ListViewLinearLayout HorizontalItemSpacing="20"
+                                                  Orientation="Horizontal" />
+                </telerik:RadListView.LayoutDefinition>
+            </telerik:RadListView>
+
+            <Button Text="Scroll Week 3 to SubItem #25"
+                    Clicked="Button_OnClicked"
+                    HorizontalOptions="End"
+                    VerticalOptions="Center"
+                    Margin="10" />
+
+        </Grid>
+    </ContentPage>
+    ```
+
+    >note Notice that we're subscribing to the inner ListView's **BindingContextChanged** event. This is used to set the `KanbanItem`'s **ListViewHost** property, which is what enables programmatical scrolling of the inner list. In a real-world project, you will want to set the with a WeakReference to avoid memory leaks, or a better approach is to wrap the DataTemplate contents into a custom control where you have more control over the lifecycle.
```

Comment:
```suggestion
> Notice that you're subscribing to the inner ListView's `BindingContextChanged` event. This is used to set the `KanbanItem`'s `ListViewHost` property, which is what enables programmatical scrolling of the inner list. In a real-world project, you will want to set the with a `WeakReference` to avoid memory leaks, or a better approach is to wrap the `DataTemplate` contents into a custom control where you have more control over the lifecycle.
```